### PR TITLE
Store preauth data sequence for renewing TGT

### PR DESCRIPTION
**Pull request by NeilGerring**
If the KDC requires preauthentication when getting a TGT, Gokrb5 will store the encryption type used for later user when renewing the TGT. Unfortunately, in some cases a pre auth data sequence needs to also be reused. This makes a change so that the data sequence can be reused.

### CODE REVIEW

File: v8/client/client.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type Client struct {
 // Set the realm to empty string to use the default realm from config.
 func NewWithPassword(username, realm, password string, krb5conf *config.Config, settings ...func(*Settings)) *Client {
 	creds := credentials.New(username, realm)
-	return &Client{
+	cl := &Client{
```
Comment (edwardzsmith) [inline v8/client/client.go]: Is this change necessary?

**Review by edwardzsmith — APPROVED** [review]
Looks ok to me - although I would be slightly concerned that adding `paData types.PADataSequence` to the settings might have an unexpected side effect somewhere else in the code.

File: v8/client/client.go (hunk 1 above)
Comment (NeilGerring) [reply v8/client/client.go]: That was a left over from when I was logging stuff out to try figure out the issue. Good spot, it's been removed
